### DISCUSSION TITLE
SMTP Disabled Error

### DISCUSSION THREAD

**Discussion by Maxbrt18** [appwrite/appwrite#4932] [Q&A] [answered]
### 👟 Reproduction steps

When I send a password recovery, I get an error (which I did not have before upgrading to 1.2.0)

### 👍 Expected behavior

It should send the email without error.

### 👎 Actual Behavior

I have the following error :
`{message: SMTP Disabled, code: 503, type: general_smtp_disabled, version: 1.2.0}`

### 🎲 Appwrite version

Version 1.2.x

### 💻 Operating system

Linux

### 🧱 Your Environment

I use latest version of Ubuntu.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment, marked as answer]
This error occurs because the `_APP_SMTP_HOST` environment variable in your `.env` file is blank. Please make sure you fill in the [SMTP environment variables](https://appwrite.io/docs/environment-variables#smtp). You can validate the env vars are set in your container by running `docker compose exec appwrite vars`.

For more information on configuring email delivery, refer to the [Email Delivery docs](https://appwrite.io/docs/email-delivery).

**Reply by Maxbrt18** [reply]
Well I just checked and it is not empty :

```
_APP_SMTP_HOST=smtp-relay.sendinblue.com
_APP_SMTP_PORT=587
_APP_SMTP_SECURE=tls
_APP_SMTP_USERNAME=...
```

**Reply by Maxbrt18** [reply]
This problem didn't exist before, so I don't think it's a configuration (or maybe a migration) problem.

**Reply by stnguyen90** [reply]
Would you please check by running `docker compose exec appwrite vars`?

**Comment by hugo-abdou** [comment]
I'm still getting the same problem with 1.5 version

![image](https://github.com/appwrite/appwrite/assets/44982282/475bba77-5fe1-4a89-9450-f1d611fa5eac)

![image](https://github.com/appwrite/appwrite/assets/44982282/cc1f0256-0a95-44ef-9de6-3f2c07590b5f)

**Comment by SeveToo** [comment]
I have the same problem

![image](https://github.com/user-attachments/assets/c36562c1-adc8-41cf-9604-0527db44cfb4)
